src/main/java/org/craftercms/core/cache/Cache.java

```diff
@@ -228,4 +228,11 @@ void put(String scope, Object key, Object value, List<Object> dependencyKeys, lo
      */
     void clearScope(String scope) throws InvalidScopeException, InternalCacheEngineException;
 
+    /**
+     * Returns the statistics for all scopes in the cache.
```

Comment:
Sorry missed this one :). Fix it to by changing to `Returns the statistics of the specified scope...`